build.rs

```diff
@@ -17,6 +23,7 @@ fn main() {
             println!("cargo:rustc-link-arg=-L/opt/homebrew/opt/libomp/lib");
```

Comment:
Why do we have duplication of the link flags ?

Reply:
what duplication?